SwiftSyft/Classes/SyftWebSocket_iOS12.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public class SyftWebSocketIOS12: SyftWebSocket, WebSocketDelegate {
+    var socket: WebSocket!
+
+    public func didReceive(event: WebSocketEvent, client: WebSocket) {
+        switch event {
+        case .connected(let headers):
+            connected()
```

Comment:
Call this in the delegate since the signalling client will be the one to handle `SocketClientDelegate` methods.
`self.delegate.connected()`

Reply:
fixed